### PR TITLE
Constants

### PR DESCRIPTION
**Pull request by aridavidson001**
adding in some constants - gear ratios for pivot, intake pivot, and elevator

### CODE REVIEW

**Comment by aridavidson001** [comment]
where should I move the trajectory constants to?

**Comment by Ishan1522** [comment]
> where should I move the trajectory constants to?

add as a sub class within SwerveConstants

**Review by Ishan1522 — APPROVED** [review]
i approve

File: src/main/java/frc/robot/subsystems/shooter/ShooterConstants.java [hunk 1]
```diff
@@ -14,6 +14,7 @@ public final class ShooterConstants {
   public static final int FOLLOWER_FLYWHEEL_ID = 12;
   public static final int ROLLER_MOTOR_ID = 2;
 
+  public static final double SHOOTER_GEARING = 1;
```
Comment (Ishan1522) [inline src/main/java/frc/robot/subsystems/shooter/ShooterConstants.java:17]: u dont need gearing if it is 1:1

File: src/main/java/frc/robot/subsystems/elevator/ElevatorConstants.java [hunk 2]
```diff
@@ -2,13 +2,14 @@
 
 public class ElevatorConstants {
   public static final double DRUM_RADIUS = 0 - 9;
-  public static final double GEAR_RATIO = 0 - 9;
-  public static final double MIN_HEIGHT = 0 - 9;
+  public static final double GEAR_RATIO = 2;
+  public static final double ENCODER_CONVERSION_FACTOR = 71.81;
```
Comment (Ishan1522) [inline src/main/java/frc/robot/subsystems/elevator/ElevatorConstants.java:6]: keep naming convention consistent to `SUBSYSTEM_GEAR_RATIO`

File: src/main/java/frc/robot/subsystems/pivot/PivotConstants.java [hunk 3]
```diff
@@ -3,9 +3,10 @@
 import com.ctre.phoenix6.signals.SensorDirectionValue;
 
 public class PivotConstants {
-  public static final double PIVOT_MASS = 0.0;
-  public static final double PIVOT_LENGTH = 0.0;
-  public static final double PIVOT_GEARING = 0.0;
+  public static final double PIVOT_MASS = 0 - 9;
+  public static final double PIVOT_LENGTH = 0 - 9;
+
+  public static final double PIVOT_GEARING = 8.0;
```
Comment (Ishan1522) [inline src/main/java/frc/robot/subsystems/pivot/PivotConstants.java:9]: keep naming convention consistent to `SUBSYSTEM_GEAR_RATIO`

**Comment by aridavidson001** [comment]
rip me not having merge authority

**Comment by Ishan1522** [comment]
Huh

**Comment by Ishan1522** [comment]
mb ig

**Pull request merged**